docs/README.md

```diff
@@ -0,0 +1,33 @@
+# Docs Directory
+
+## How-Tos
+* [Dev Of The Week Zendesk](dev-of-the-week-zendesk.md) - How to find stuff in zendesk if you are dev of the week
+* [Dropbox Share Guide](dropbox-share-guide.md) - How to add new dropbox shares to staging
+* [Honeybadger](honeybadger.md) - How to use honeybadger to triage site issues
+* [Importing Data into Pegasus](importing-data.md)
+* [Instrumenting with Google Analytics](instrumenting-analytics.md)
+* [Logging](logging.md) - How to do various things with dashboard and pegasus logs
+* [Where Are The Logs](where-are-the-logs.md) - Some more information about logging
+* [Pdf Lesson Plan Generation](pdf-lesson-plan-generation.md) - how to create pdf lesson plans from markdown files
+* [Pegasus Dashboard Integration](pegasus-dashboard-integration.md) - how to access dashboard data from pegasus
+* [Reset Password Manually](reset-password-manually.md) - how to reset user passwords
+* [Testing Production Locally](testing-production-locally.md) - how to run your local server like production
+* [Testing With Applitools Eyes](testing-with-applitools-eyes.md) - how to write and run ui tests that will have their screenshots compared
+* [Update Levelbuilder](update-levelbuilder.md) - how to update levelbuilder
+* [Varnish](varnish.md) - info on our use of varnish and how to work with it
+
+## Technical Specifications
+* [Account Management API](account-management-api.md)
+* [Client API](client-api.md)
```

Comment:
Yes, it is worth keeping around, and also needs updating. I wonder how much other documentation we should have in a place that's visible to contributors, instead of in the wiki.